#! /bin/sh
patch -p1 -f $* < $0
exit $?

commit fc3ae4dc1065a1cc42a0e3ea0f056a8eb7da928d
Author: Andrej Kacian <ticho@claws-mail.org>
Date:   Sat Dec 29 13:19:30 2018 +0100

    Fix alertpanel API use in win32 part of mimeview.c

diff --git a/src/common/codeconv.h b/src/common/codeconv.h
index 270498558..862d303dd 100644
--- a/src/common/codeconv.h
+++ b/src/common/codeconv.h
@@ -178,6 +178,11 @@ struct _CodeConverter
 #define C_INTERNAL		C_UTF_8
 #define CS_INTERNAL		CS_UTF_8
 
+typedef enum {
+	CODECONV_NOSPACE = -2,
+	CODECONV_FAIL = -1,
+	CODECONV_OK   = 0
+} CodeconvResult;
 
 void conv_utf8todisp	(gchar *outbuf, gint outlen, const gchar *inbuf);
 void conv_localetodisp	(gchar *outbuf, gint outlen, const gchar *inbuf);
diff --git a/src/mimeview.c b/src/mimeview.c
index b7068eaf1..daaac32d4 100644
--- a/src/mimeview.c
+++ b/src/mimeview.c
@@ -2294,7 +2294,7 @@ static void mimeview_view_file(const gchar *filename, MimeInfo *partinfo,
 					"your computer.\n\n"
 					"Do you want to run this file?"), GTK_STOCK_CANCEL, 
 					_("Run binary"),
-		      		      NULL, FALSE, NULL, ALERT_WARNING, G_ALERTDEFAULT);
+		      		      NULL, ALERTFOCUS_FIRST, FALSE, NULL, ALERT_WARNING);
 		if (val == G_ALERTALTERNATE) {
 			debug_print("executing binary\n");
 			ShellExecute(NULL, L"open", (LPCWSTR)fn16, NULL, NULL, SW_SHOW);
